README.md

```diff
@@ -25,6 +25,10 @@
 | PR_DATABASE_URL             | jdbc:postgresql://host:port/database   | database url to connect to your database                              |
 | PR_DATABASE_USER            | user                                   | database user                                                         |
 | PR_DATABASE_PASSWORD        | super-secret-password                  | database password                                                     |
+| PR_SMTP_SERVER              | smtp.gmail.com                         | smtp server of your email provider                                    |
+| PR_SMTP_PORT                | 587                                    | smtp server port                                                      |
+| PR_SMTP_EMAIL               | example@gmail.com                      | your email address of your email provider                             |
+| PR_SMTP_APP_PW              | add****************nfs                 | your application password of your email                               |
```

Comment:
application password is specific to google. In general it is just a normal password for the smtp server.

Reply:
changed the env. 
but already pushed something on this branch. I will merge it now. If the env is still wrong, just contact me.